addons/mail/models/mail_rotting_resource_mixin.py

```diff
@@ -0,0 +1,55 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+from datetime import timedelta
+
+
+class RottingResourceMixin(models.AbstractModel):
+    _name = 'mail.rotting.resource.mixin'
```

Comment:
Would say to include in 'mail.tracking.duration.mixin', as we have at lest 3 common models (lead, applicant, task), and we are going to share some tools (find stage notably). I bet it is going to be useful on tickets anyway; maybe just projects won't use it but hey, they will just have an extra field and tant pis.

Reply:
Also I would strongly advise adding tests in test_mail: create a new model (or re-use a test model inheriting from mail tracking duration mixin). It is going to help you I bet.